utils.hpp

```diff
@@ -55,16 +55,7 @@ constexpr auto digits_amount(int x)
 {
     x = abs(x);
 
-    return (x < 10 ? 1 :   
-        (x < 100 ? 2 :   
-        (x < 1000 ? 3 :   
-        (x < 10000 ? 4 :   
-        (x < 100000 ? 5 :   
-        (x < 1000000 ? 6 :   
-        (x < 10000000 ? 7 :  
-        (x < 100000000 ? 8 :  
-        (x < 1000000000 ? 9 :  
-        10)))))))));  
+    return (x < 10 ? 1 : (x < 100 ? 2 : (x < 1000 ? 3 : (x < 10000 ? 4 : (x < 100000 ? 5 : (x < 1000000 ? 6 : (x < 10000000 ? 7 : (x < 100000000 ? 8 : (x < 1000000000 ? 9 : 10)))))))));
```

Comment:
I could probably disable clang-tidy just for these lignes.

Reply:
I agree, the original layout is more readable.